common/djangoapps/enrollment/data.py

```diff
@@ -16,8 +16,13 @@
     InvalidEnrollmentAttribute,
     UserNotFoundError
 )
-from enrollment.serializers import CourseEnrollmentSerializer, CourseSerializer
+from enrollment.serializers import (
```

Comment:
```suggestion
from enrollment.serializers import CourseEnrollmentSerializer, CourseSerializer
from enrollment.serializers import EdraakCourseEnrollmentSerializer
```

Better for git conflicts, unless `pylint` don't like it.